app/models/grda_warehouse/system_cohorts/chronic_individual.rb

```diff
@@ -0,0 +1,24 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module GrdaWarehouse::SystemCohorts
+  class ChronicIndividual < CurrentlyHomeless
+    include ArelHelper
+
+    def cohort_name
+      'Chronic Individual'
+    end
+
+    private def enrollment_source
+      GrdaWarehouse::ServiceHistoryEnrollment.
+        entry.ongoing(on_date: @processing_date).
+        where(
+          presented_as_individual: true,
```

Comment:
Hmm, I'm not sure I want to use this criteria.  Can we adjust to "Adult Only Household".  I don't think the intention is that there is only one person in the household, but rather to avoid including children.   That'll probably make this more complicated (find open chronic enrollments, collect up household members, throw out enrollments with children...)